zephyr-apps/gpio/sample.yaml

```diff
@@ -0,0 +1,10 @@
+sample:
+  name: Blinky Sample
+tests:
+  sample.basic.blinky:
+    tags: LED gpio
+    filter: dt_enabled_alias_with_parent_compat("led0", "gpio-leds")
+    depends_on: gpio
+    harness: led
+    integration_platforms:
+      - frdm_k64f
```

Comment:
You can just removed this file, its provided in the sample code but its not need.